Extensions/quick_tags.js

```diff
@@ -273,6 +279,18 @@ XKit.extensions.quick_tags = new Object({
 
 	},
 
+	render_add_bundle_button: function(for_one_click) {
+		var m_add_button = "";
+
+		if (user_tag_array.length >= 30 && 
+			for_one_click === true &&
```

Comment:
If we're always going to do nothing when `for_one_click` is passed, do we even need the parameter or can we just leave the function call out entirely?